docs/transpile/customizing-transpilation.ipynb

```diff
@@ -0,0 +1,414 @@
+{
```

Comment:
<p>could you explain here what the difference is between a regular pass manager and a staged pass manager?</p><p> </p><p>Also, I feel like maybe this section should go above the "create your own passmanager section", what do you think? This part seems significantly easier than the creating your own passmanger section, and I think it's generally best to order the page with easiest stuff first</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/391/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
Done.